src/dlgAliasMainArea.cpp

```diff
@@ -27,3 +27,13 @@ dlgAliasMainArea::dlgAliasMainArea(QWidget* pF) : QWidget(pF)
     // init generated dialog
     setupUi(this);
 }
+
+void dlgAliasMainArea::trimName()
+{
+    lineEdit_alias_name->setText(lineEdit_alias_name->text().trimmed());
+}
+
+void dlgAliasMainArea::on_lineEdit_alias_name_editingFinished()
```

Comment:
We like to use a `slot_` prefix for slot functions ([example](https://github.com/Mudlet/Mudlet/blob/development/src/mudlet.cpp#L411)), so try changing it to something like `slot_name_editing_finished`.

Also you've gotta hook the slot up to the signal, so add the following off the top of my head:

```
connect(lineEdit_alias_name, &QLineEdit::editingFinished, this, slot_name_editing_finished);
```

Reply:
Cool, I know I was gonna mess something up adding to an empty file.

On a side note, the saveSomething() functions are not called when you save the profile, is it intended?

Reply:
We can make it call them, sure. Another PR? :smile: